overview pages for BTC staking / timestamping (#35)
babylonchain/babylonchain.github.io: docs/introduction/babylon-overview.md

# Babylon Overview

<!-- Babylon vision -->

> Babylon is a project that designs security protocols for the decentralized world.
> Our mission is to scale Bitcoin to secure the decentralized world.

Reviewer: ```suggestion
Babylon is a project that designs security protocols that aim to scale Bitcoin to secure the decentralized world.
```

> To this end, Babylon utilizes the three primary facets of Bitcoin: Bitcoin as an asset, its reliable timestamping service, its most censorship-resistant blockspace in the world.

Reviewer: ```suggestion
To this end, Babylon utilizes the three primary facets of Bitcoin: Bitcoin the asset, Bitcoin the reliable timestamping server, and Bitcoin the blockchain that contains the most censorship-resistant blockspace in the decentralized world.
```

Through inventing pioneering protocols, Babylon envisions a more secure and decentralized future.

<!-- Two protocols in Babylon. BTC timestamping is a building block for BTC staking -->
We build the [Babylon](https://github.com/babylonchain/babylon) blockchain to realize the vision.

> It is a proof-of-stake (PoS) blockchain that consists of two innovative security-sharing protocols bridging Bitcoin and other PoS blockchains:

Reviewer: ```suggestion
It is a proof-of-stake (PoS) blockchain that consists of two innovative security-sharing protocols connecting Bitcoin and other PoS blockchains:
```

might want to avoid the word "bridge" as it carries some negative weight and might imply that we are a bridging solution.

- **Bitcoin timestamping:** This protocol sends succinct and verifiable timestamps of PoS blockchains to Bitcoin; and

> - **Bitcoin staking:** Using Bitcoin timestamping as a building block, this protocol allows Bitcoin holders to stake their Bitcoin to a PoS blockchain.

Reviewer: ```suggestion
- **Bitcoin staking:** Using Bitcoin timestamping as a building block, this protocol allows Bitcoin holders to stake their Bitcoin to PoS blockchains.
```
